### ISSUE TITLE
Jaws: Ensure they cannot crash

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#2524]
It appears that I can set a negative gap into the jaws IOC, this causes the underlying motors to be set in such a way so that they may potentially collide. 

**Acceptance Criteria**
* I cannot set a negative gap
* The Jaws IOC will go into alarm if there is a negative gap
* I cannot set the individual motors in such a way that the jaws will crash (at least through the aliases on the Jaws OPI)

**Comment by KathrynBaker** [comment]
This is a change we can’t make, as a negative gap is actually possible on some jaw sets. There are some which can overlap for most of their travel, and even report a lower limit based on the travel in relation to each other. Whilst unusual I’m sure there are some reasons for giving that negative value, rather than 0 – e.g. if the homing routines are not quite accurate enough and you know you want your jaws closed, you overlap them to ensure no gap remains.

This is probably something we have to allow.

**Comment by DominicOram** [comment]
In view of that we should instead be able to set a variable lower limit, maybe as a macro.

**Comment by KathrynBaker** [comment]
That sounds like a very good idea.
